docker/build.mk

```diff
@@ -184,33 +184,44 @@ $(foreach fuzzer,$(FUZZERS), \
 
 
 define oss_fuzz_benchmark_template
-$(1)-project-name := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
-                             grep project | cut -d ':' -f2 | tr -d ' ')
+# Use eval to make variables accessible within this template.
+$(eval $(1)-project-name := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
+                             grep project | cut -d ':' -f2 | tr -d ' '))
 $(1)-fuzz-target  := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
                              grep fuzz_target | cut -d ':' -f2 | tr -d ' ')
-$(1)-oss-fuzz-builder-hash := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
+$(eval $(1)-oss-fuzz-builder-hash := $(shell cat benchmarks/$(1)/oss-fuzz.yaml | \
                                       grep oss_fuzz_builder_hash | \
-                                      cut -d ':' -f2 | tr -d ' ')
+                                      cut -d ':' -f2 | tr -d ' '))
+.$(1)-fuzzbench-builder:
```

Comment:
The name "fuzzbench" might be misleading. How about something like "oss-fuzz-wrapper"?